src/direct-binary-upload-process.js

```diff
@@ -81,6 +83,19 @@ export default class DirectBinaryUploadProcess extends FileTransferHandler {
         this.fileEvents = {};
         this.fileTransfer = {};
         this.completeUri = '';
+
+        const log = options.log;
+        if (log) {
+            httpTransferLogger.debug.log = (...args) => log.debug.apply(log, args);
+            httpTransferLogger.info.log = (...args) => log.info.apply(log, args);
+            httpTransferLogger.warn.log = (...args) => log.warn.apply(log, args);
+            httpTransferLogger.error.log = (...args) => log.error.apply(log, args);
+            // debug.enable("httptransfer:*");
+        }
+
+        httpTransferLogger.info("test info");
+        httpTransferLogger.warn("test warn");
+        httpTransferLogger.error("test error");
```

Comment:
Are these "debugging" messages that should be removed?

Reply:
I've removed them